.github/workflows/linter.yml

```diff
@@ -1,60 +1,85 @@
-    # https://help.github.com/en/articles/workflow-syntax-for-github-actions
-    #
-    
-    #############################
-    # Start the job on all pull requests #
-    #############################
-    on:
-      pull_request_target:
-    
-    ###############
-    # Set the Job #
-    ###############
-    jobs:
-      build:
-        # Name the Job
-        name: Lint Code Base
-        # Set the agent to run on
-        runs-on: ubuntu-latest
-    
-        ##################
-        # Load all steps #
-        ##################
-        steps:
-          ##########################
-          # Checkout the code base #
-          ##########################
-          - name: Checkout Code
-            uses: actions/checkout@v2
-
-          ##########################
-          # Github Super Linter needs
-          # the latest definitions installed
-          ##########################
-          - name: Use Node.js 16.x
-            uses: actions/setup-node@v2.5.1
-            with:
-              node-version: 16.x
-          - run: npm install
-
-          ################################
-          # Run Linter against code base #
-          ################################
-          - name: Lint Code Base
-            uses: docker://ghcr.io/github/super-linter:slim-v4
-            env:
-              DEFAULT_BRANCH: main
-              LINTER_RULES_PATH: /
-              VALIDATE_ALL_CODEBASE: true
-              TYPESCRIPT_ES_CONFIG_FILE: .eslintrc.js
-              VALIDATE_TYPESCRIPT_STANDARD: false
\ No newline at end of file
+    # Set the agent to run on
+    runs-on: ubuntu-latest
+
+    ##################
+    # Load all steps #
+    ##################
+    steps:
+      ##########################
+      # Checkout the code base #
+      ##########################
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      ##########################
+      # Github Super Linter needs
+      # the latest definitions installed
+      ##########################
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v2.5.1
+        with:
+          node-version: 16.x
+      # Install our eslint packages.
+      # We may have custom tsconfigs, eslints that are brought in via a package.
+      - run: npm install
+
+      ################################
+      # Run Linter against code base #
+      ################################
+      - name: Lint Code Base
+
+        # We use the Github super linter, it can be cranky at times, so in that moment here are the docs https://github.com/github/super-linter
+        uses: docker://ghcr.io/github/super-linter:slim-v4
+
+        # All Environment variables are defined here https://github.com/github/super-linter#environment-variables
+        env:
+          # The name of the repository default branch.
+          DEFAULT_BRANCH: main
+
+          # Directory for all linter configuration rules. We symlink our root .eslintrc.js to this directory
```

Comment:
one ask - what directory is this pointing to in the code base? it's `.github/linters/.eslintrc.js`, right? can we put that in a comment here?

if the above is correct, it's really confusing of github linter to equate `/` with `/linters`.

Reply:
So!! It used to be. They updated it. That is now the root of the code. I also deleted the associated linters directory.